.storybook/main.js

```diff
@@ -24,7 +24,7 @@ module.exports = {
                 { loader: require.resolve('less-loader') },
             ],
         });
-        config.resolve.extensions.push('.ts', '.tsx');
+        config.resolve.extensions.push('.ts', '.tsx', '.less');
```

Comment:
Måtte ha denne for å lage storybook av Header ettersom eksterne avhengigheter bruker less-import uten `.less` i import-statementet